tbx/static_src/sass/components/_tag.scss

```diff
@@ -0,0 +1,32 @@
+@use "config" as *;
+
+.tags {
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 5px;
+    row-gap: 15px;
+}
+
+.tag {
+    padding: 5px $spacer-mini;
+    border-radius: 50px;
+    border: 1px solid var(--color--text);
+
+    &:hover,
+    &:active {
+        color: var(--color--page);
+        background-color: var(--color--heading);
+        border-color: var(--color--heading);
+
+        @include high-contrast-mode() {
+            &:hover,
+            &:active {
+                outline: 2px solid var(--color--heading);
+            }
+        }
```

Comment:
Q: should the `:hover` and `:active` styles here be moved outside of the global `:hover` and `&:active` styles that start on line 15 & 16 otherwise I think you'll have `.tag:hover:hover`

Reply:
oh good spot you're right.

Reply:
Fixed in 5673b7761dadb153ae209c355cdfc242a2e345ff